addons/mail/static/src/core/common/composer.dark.scss

```diff
@@ -1,3 +1,7 @@
-.o-mail-Composer-actions button:hover {
-    background-color: $gray-300;
+.o-mail-Composer-actions button {
+    @media (hover: hover) {
```

Comment:
So we should wrap all `:hover` calls in this media all the time?

Reply:
Smartphone are not `@media (hover)` but they could still put `:hover` on elements, notably after click on button.
I think this doesn't make sense, but that's how browsers behave. I think bootstrap didn't take that into account in their style, because the behaviour seems specifically problematic in mobile.